scrapers/Babepedia.yml

```diff
@@ -56,6 +56,17 @@ xPathScrapers:
               - regex: (\w+:)(\s)(\w+)(\s)(\d+)(\w+)(\s)(\w+)(\s)(\w+)(\s)(\d+)
                 with: $5 $10 $12
           - parseDate: 2 January 2006
+      DeathDate:
+        selector: $label[text()='Died:']]/text()
+        postProcess:
+          - replace:
+              - regex: st\s|nd\s|rd\s|th\s
+                with: " "
```

Comment:
The above replaces the `st` from August eg `https://www.babepedia.com/babe/Victoria_Paris`
I would use the below instead
```yaml
              - regex: (\d+)(st|nd|rd|th)
                with: "$1"
```